e2e_tests/helpers.py

```diff
@@ -33,28 +33,31 @@ def get_auth_header(token: str) -> dict:
     return {'Authorization': f'Bearer {token}'}
 
 
+def get_full_endpoint(endpoint: str) -> str:
+    full_endpoint = ""
+    if (config.TRE_URL != ""):
+        full_endpoint = f"{config.TRE_URL}{endpoint}"
+    else:
+        full_endpoint = f"https://{config.TRE_ID}.{config.RESOURCE_LOCATION}.cloudapp.azure.com{endpoint}"
+    LOGGER.info(f'POSTING RESOURCE TO: {full_endpoint}')
```

Comment:
Remove this log line? This method can be called not only for posting a resource. Then you won't need to have the variable `full_endpoint` at all